apps/teams-test-app/src/components/Links.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { ReactElement } from 'react';
+
+import { ApiWithTextInput } from './utils';
+
+const MoveToLink = (): ReactElement =>
+  ApiWithTextInput<string>({
+    name: 'moveToLink',
+    title: 'Move to Link',
+    onClick: {
+      validateInput: input => {
+        if (typeof input !== 'string') {
+          throw new Error('Input should be a string');
+        }
+      },
+      submit: async input => {
+        window.location.href = input;
+        return Promise.resolve('Moved to new link');
+      },
+    },
+  });
+
+const OpenLinkInNewWindow = (): ReactElement =>
+  ApiWithTextInput<string>({
+    name: 'openLinkInNewWindow',
+    title: 'Open link in new window',
+    onClick: {
+      validateInput: input => {
+        if (typeof input !== 'string') {
+          throw new Error('Input should be a string');
+        }
+      },
+      submit: async input => {
+        window.open(input);
+        return Promise.resolve('Link opened');
+      },
+    },
+  });
+
+const Links = (): ReactElement => (
+  <>
+    <h1>Links</h1>
```

Comment:
Not sure if we should specially call out somewhere that this is not APIs  and instead they are windows interface related. Maybe people will know by reading app.tsx since it is different from other *callName*API.